### ISSUE TITLE
Payment Gateway's

### ISSUE THREAD

**Issue by jzmwebdevelopment** [firesale/firesale#17]
Adding to more Payment Gateways:

Bank Transfer

Account Name:

Account Number:

Cheque/Think Americans call it money order?

Account Name:
Post Address:

**Comment by jHoldroyd** [comment]
I've assigned this one to Chris but i'm not sure where it'll come in our current list of priorities or when he'll have a chance to work on it but we'll discuss it tomorrow and update you.

**Comment by jzmwebdevelopment** [comment]
Thanks Guys,

If this could get done that would be great I had a look but good not 
work out where the data was being fed from.

I have a use for this if it can be done.

**Comment by jHoldroyd** [comment]
It comes from three places if I remember correctly (that's Chris' area) - there's a class under libraries for each gateway as well as an entry im the gateway language file. You'll also need to add a view containing the required fields that's shown during checkout.

Once you have those you can go into the admin area for gateways and it'll be shown under install new gateway. You should then be able to configure the required options.

As far as I'm aware it's not meant to be that difficult but everything is powered by a slightly modified version of a codeigniter library so you should probably start there to see if anyone has already done what you're after.

Sent from my Windows Phone

**Comment by chrisnharvey** [comment]
The data for the gateways comes from the CI-Merchant driver (each gateway has its own class in libraries/payments) and on the frontend there is a view for each gateway (with the same name as the driver class).

Will add a bank transfer gateway when I get a chance.

**Comment by yllumi** [comment]
I agree if bank transfer gateway become a priority, because this payment model still become a major in Indonesia and maybe for some other growing country. :D

**Comment by jHoldroyd** [comment]
We're still looking at our options and I think this will be added in 1.2 when we replace the version of ci-merchant that we're using with the latest one that already has something like this in place.

**Comment by chrisnharvey** [comment]
Moved this one to 1.2, but will be earlier than that. We need to move the order statuses from numbers to 'slugs', then we can add a 'pending' order status.
